### PR TITLE
fix make clean

### PR DESCRIPTION
**Pull request by fdr10**
## What it does ?
fixes error when running the clean target in the top level Makefile
## How it does it ?
In the top level Makefile replace `cd aurae && make clean` with `cd auraescript && make clean`

### CODE REVIEW

**Comment by fdr10** [comment]
@cla-bot check

**Comment by fdr10** [comment]
Hello ! This is my first open source pr. Looking forward to contribute more as I get familiar with the project. Starting small. 🙂

File: Makefile [hunk 1]
```diff
@@ -115,8 +115,8 @@ fmt: headers ## Format the entire code base(s)
 
 .PHONY: clean
 clean: clean-certs
-	cd aurae && make clean
 	cd auraed && make clean
+	cd auraescript && make clean
 	@rm -rvf target/*
```
Comment (dominichamon) [inline Makefile:120]: this is ok, but i wonder if we should _also_ call `cargo clean` as we do in the sub-folder Makefiles.

Comment (krisnova) [reply Makefile:120]: What does the nested makefile do? Doesn't that itself call cargo clean?

Comment (fdr10) [reply Makefile]: Hi everyone 🙂. Yeah they call `cargo clean`.  This got me thinking if instead we should replace the clean target with a `cargo clean` (which also does an `rm -rf /target`)   while at the same time leverage the existing Makefiles to clean the packages (auraed and auraescript) individually if needed.

Comment (krisnova) [reply Makefile]: Sure however you want to do it. I will merge now.

**Review by krisnova — APPROVED** [review]

**Pull request merged**